.github/workflows/node-ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Use Node.js 12.x
         uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 12.16.x
```

Comment:
I changed this b/c storybook/chromatic was failing to build: https://github.com/storybookjs/storybook/issues/10477#issuecomment-635075061

Reply:
Hopefully they'll fix the underlying issue in the future.